src/stirling/source_connectors/dynamic_bpftrace/dynamic_bpftrace_connector_bpf_test.cc

```diff
@@ -212,19 +219,19 @@ TEST(DynamicBPFTraceConnectorTest, BPFTraceBuiltins) {
 
     int64_t nsecs = records[kNsecsIdx]->Get<types::Int64Value>(0).val;
     LOG(INFO) << absl::Substitute("nsecs: $0", nsecs);
-    // Event should have been in the last 10 seconds (being generous).
+    // Check that the timestamp is after the connector was initialized, and before the current
+    // timestamp.
     auto now = px::chrono::boot_clock::now().time_since_epoch();
     auto now_ns = std::chrono::duration_cast<std::chrono::nanoseconds>(now).count();
-    constexpr int64_t kNanosPerSecond = 1000 * 1000 * 1000;
-    EXPECT_GE(nsecs, now_ns - 10 * kNanosPerSecond);
+    EXPECT_GE(nsecs, deploy_start_ns);
     EXPECT_LE(nsecs, now_ns);
 
     int64_t elapsed = records[kElapsedIdx]->Get<types::Int64Value>(0).val;
     LOG(INFO) << absl::Substitute("elapsed: $0", elapsed);
-    // Expecting a value of close to 100 ms. Add +/- 50 ms to be generous.
+    // Expecting a value of close to 100 ms. Look for between 50 and 200 to be generous.
     constexpr int kNanosPerMilli = 1000 * 1000;
     EXPECT_GE(elapsed, 50 * kNanosPerMilli);
-    EXPECT_LE(elapsed, 150 * kNanosPerMilli);
+    EXPECT_LE(elapsed, 200 * kNanosPerMilli);
```

Comment:
This seems liable to cause flakes in the future, do we really need to have an expectation for the elapsed time?